src/core/Tokenizer.js

```diff
@@ -12,15 +12,17 @@ export default class Tokenizer {
      *  @param {String[]} cfg.closeParens Closing parentheses to enable, like ), ]
      *  @param {String[]} cfg.indexedPlaceholderTypes Prefixes for indexed placeholders, like ?
      *  @param {String[]} cfg.namedPlaceholderTypes Prefixes for named placeholders, like @ and :
+     *  @param {String[]} cfg.lineCommentTypes Line comments to enable, like # and --
      */
     constructor(cfg) {
-        this.WORD_REGEX = /^(\w+)/;
+        this.WORD_REGEX = /^([\w|#|@]+)/;
```

Comment:
We should also make this WORD_REGEX configurable. We could have a config option like `specialWordChars: ["@", "#"]`.

Other SQL dialects don't allow `#` and `@` inside identifiers, which currently results in a scenario where formatting the following query with StandardSqlFormatter:

```sql
SELECT a#comment, here
```

results in:

```sql
SELECT
    a#comment,
    here
```

while correct formatting would be:

```sql
SELECT
    a #comment, here
```